stock_available/views/res_config_view.xml

```diff
@@ -22,6 +22,8 @@
                                 <div>
                                     <field name="module_stock_available_mrp" class="oe_inline" />
                                     <label for="module_stock_available_mrp" />
+                                    <label for="stock_available_mrp_based_on" />
+                                    <field name="stock_available_mrp_based_on" class="oe_inline" attrs="{'required':[('module_stock_available_mrp','=',True)]}"/>
```

Comment:
Yes, there is a reason :)

If you set the option in stock_available_mrp, you should first install stock_available_mrp (check the box and valid) and in a second step you can configure the stock field. If the option is in stock_available, you can directly configure all you need in one step.